### DISCUSSION TITLE
Connect Tilt straight to ESP32 flashed controller

### DISCUSSION THREAD

**Discussion by jon-peachey** [thorrak/brewpi-esp#60] [Ideas]
Can the tilt be connected to the esp32 Bluetooth connection, instead of connecting to the pi or a tiltbridge (second esp32)?

**Comment by thorrak** [comment]
Yep, working on it.

**Reply by jon-peachey** [reply]
Thanks. and for the quick reply! love your work!

**Comment by thorrak** [comment]
It took a bit, but the v15 firmware now supports using a Tilt as a temperature sensor.
